src/test/kotlin/io/openfuture/state/controller/WalletControllerTest.kt

```diff
@@ -66,4 +71,10 @@ class WalletControllerTest : ControllerTests() {
                     }
                 """.trimIndent())
     }
+
+    private fun createDummySaveWalletRequest(
+            address: String = "address",
+            webhook: String = "webhook",
+            blockchain: Blockchain = createDummyBlockchain()
```

Comment:
One more time: You should not create a dummy blockchain object. You should mock it instead.

Reply:
This specific object can have a String type for the `blockchain` property as it's a user request